tests/integration/remote/invoke/remote_invoke_integ_base.py

```diff
@@ -0,0 +1,105 @@
+from unittest import TestCase, skipIf
+from pathlib import Path
+from typing import Optional
+
+from tests.testing_utils import (
+    get_sam_command,
+    run_command,
+    RUNNING_ON_CI,
+    RUNNING_TEST_FOR_MASTER_ON_CI,
+    RUN_BY_CANARY,
+)
+from tests.integration.deploy.deploy_integ_base import DeployIntegBase
+
+from samcli.lib.utils.boto_utils import get_boto_resource_provider_with_config, get_boto_client_provider_with_config
+from samcli.lib.utils.cloudformation import get_resource_summaries
+
+SKIP_REMOTE_INVOKE_TESTS = RUNNING_ON_CI and RUNNING_TEST_FOR_MASTER_ON_CI and not RUN_BY_CANARY
```

Comment:
why do we need to skip running the remote test cases in case of master branch?

Reply:
Seems like this check might not be required anymore since we moved the AppVeyor tests that run on each PR to GH actions. I can remove this for `remote invoke` tests first and update the other commands in a separate PR.